docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -433,17 +570,14 @@ Menelusuri dokumentasi dalam Bahasa Portugis Brasil akan menghasilkan sidebar be
 
 Grup-grup tautan dapat dikembangkan secara default dengan mengatur properti `collapsed` menjadi `true`.
 
-```js
+```js {5-6}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
-			// Kembangkan grup secara default.
+			label: 'Constellations',
```

Comment:
The label can be translated in this code block and in the example below.

Reply:
Ok, will update soon.